### PR TITLE
2 functions(not called) added to js file

### PR DESCRIPTION
**Pull request by Seanmcdon**
function 1 - creates an array of objects from the data where the key is the corresponding value from the first line of the csv file. 

function 2 - draws the table if this is the case.

function #1 - processDataAsObj()
function #2 - drawOutputAsObj()

### CODE REVIEW

File: js/read-csv.js [hunk 1]
```diff
@@ -32,6 +32,26 @@ function processData(csv) {
...
+    while (allTextLines.length) {
+        var arr = allTextLines.shift().split(',');
+        var obj = {};
+        for(var i = 0; i < keys.length; i++){
+            obj[keys[i]] = arr[i];
+	}
+        lines.push(obj);
+    }
+        console.log(lines);
+	//drawOutput(lines);
```
Comment (MounirMesselmeni) [inline js/read-csv.js]: @Seanmcdon I think you should call `drawOutputAsObj` here and remove the commented line

**Comment by MounirMesselmeni** [comment]
@Seanmcdon It would be better if you will [squash](https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/) your commits into one single.
Thanks for your contribution!

**Comment by Seanmcdon** [comment]
Ok - thank you! 
I don't think I can squash from the browser ... Sorry ... is there anything else I can do?

**Comment by Seanmcdon** [comment]
I don't think I can squash them from the browser... This will be my first
"pull" and accept if it works out - thanks for replying today! Let me know
if there's anything I can do to make up for the failed squash...

Sean

**Comment by Seanmcdon** [comment]
I can start a new one that has only one commit.

**Pull request closed without merging**